packages/connectivity/lib/connectivity.dart

```diff
@@ -14,6 +14,24 @@ import 'package:meta/meta.dart';
 /// None: Device not connected to any network
 enum ConnectivityResult { wifi, mobile, none }
 
+/// Metered: Device connected via metered Wi-Fi (Android only)
```

Comment:
put docs on each option above the option itself.
e.g
```
/// Device connected via metered Wi-Fi (Android only)
metered
```
Also, add doc to explain this enum.